airbyte-integrations/connectors/source-solarwinds-service-desk/manifest.yaml

```diff
@@ -0,0 +1,6849 @@
+version: 5.12.0
+
+type: DeclarativeSource
+
+description: "Documentation: https://apidoc.samanage.com/#section/General-Concepts"
+
+check:
+  type: CheckStream
+  stream_names:
+    - hardwares
+
+definitions:
+  streams:
+    hardwares:
```

Comment:
It doesn't look `hardwares` and other endpoints support incremental (see [docs](https://apidoc.samanage.com/#tag/Hardware/operation/getHardwares)) can you share where did you found info about saying this endpoint support injecting the dates to filter data?